stylix/darwin/default.nix

```
@@ -9,17 +14,27 @@ let
   autoload = import ../autoload.nix { inherit lib inputs; } "darwin";
 in
 {
-  imports = [
-    "${inputs.self}/stylix/darwin/fonts.nix"
-    "${inputs.self}/stylix/darwin/palette.nix"
-    "${inputs.self}/stylix/fonts.nix"
-    "${inputs.self}/stylix/home-manager-integration.nix"
-    "${inputs.self}/stylix/opacity.nix"
-    "${inputs.self}/stylix/palette.nix"
-    "${inputs.self}/stylix/pixel.nix"
-    "${inputs.self}/stylix/target.nix"
-    "${inputs.self}/stylix/release.nix"
-  ] ++ autoload;
+  imports =
+    [
+      "${inputs.self}/stylix/darwin/fonts.nix"
+      "${inputs.self}/stylix/darwin/palette.nix"
+      "${inputs.self}/stylix/fonts.nix"
+      "${inputs.self}/stylix/home-manager-integration.nix"
+      "${inputs.self}/stylix/opacity.nix"
+      "${inputs.self}/stylix/palette.nix"
+      "${inputs.self}/stylix/pixel.nix"
+      "${inputs.self}/stylix/target.nix"
+      "${inputs.self}/stylix/release.nix"
+    ]
+    ++ autoload
+    ++ (import ../overlays.nix {
```

To align with commit e87bf16df961 ("stylix: use absolute paths for module imports"), apply the following change also to the other relevant files:


```suggestion
    ++ (import "${inputs.self}/stylix/overlays.nix" {
```